src/adapter/src/catalog.rs

```diff
@@ -6603,6 +6623,12 @@ impl Catalog {
                     }
                     ObjectId::Role(_) => unreachable!("roles have no owner"),
                 },
+                Op::UpdateClusterConfig { id, config } => {
+                    let cluster = state.get_cluster_mut(id);
+                    cluster.config = config;
+                    tx.update_cluster(id, cluster)?;
+                    info!("update cluster {}", cluster.name);
```

Comment:
Do we store the config in any builtin table anywhere? If so, we'll have to update that table as well.

Reply:
Done.